<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>716</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="commentsList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>761</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="url">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>56</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>URL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startBtn">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>시작</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editUrl">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="editComment">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>600</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="commentLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>610</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>댓글</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submitBtn">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>600</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>등록</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="commentProgress">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>560</y>
      <width>661</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="loading">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>560</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Loading</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>785</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
